bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -52,7 +52,7 @@ spec:
           }
 
           # Check to see if ovs is provided by the node:
-          if [[ -L '/host/etc/systemd/system/network-online.target.wants/ovs-configuration.service' ]]; then
+          if [ -f /host/var/run/ovs-config-executed ]; then
```

Comment:
Where does this file gets generated? I don't see it being created by MCO

Reply:
https://github.com/openshift/machine-config-operator/pull/2140

only in release-4.6 branch
https://github.com/openshift/machine-config-operator/blob/release-4.6/templates/common/_base/files/configure-ovs-network.yaml#L9

in 4.7 we should only run host OVS, so I think the plan is to remove this `if`.